tools/RAiDER/runProgram.py

```diff
@@ -178,8 +178,10 @@ def parseCMD():
             args_copy.insert(idxH, hydroNames.tolist())
             lst_new_args.append(args_copy)
 
-        with multiprocessing.Pool(len(lst_new_args)) as pool:
-            pool.map(_tropo_delay, lst_new_args)
+        for arg in lst_new_args:
+            _tropo_delay(arg)
+        #with multiprocessing.Pool(len(lst_new_args)) as pool:
+        #    pool.map(_tropo_delay, lst_new_args)
```

Comment:
suggest we do a try except for now.
As otherwise we need to fix this for the files we are actively downloading for now.
I can add that in, meanwhile we can look what is wrong. 
Changing to passing of a dictionary with arguments will be much easier as well.